web/src/scss/components/watches/WatchConfig.scss

```diff
@@ -4,6 +4,7 @@
   width: 100%;
   max-width: 750px;
   min-width: 500px;
+  padding-top: 20px;
```

Comment:
This pushes the config form section down so that it's vertically aligned with the nav when the page is loaded initially.